sklearn/multioutput.py

```diff
@@ -37,12 +39,89 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.deepcopy(estimator)
```

Comment:
Yes, you will be partial-fitting one estimator per process right? Unless you are using the same estimator in 2 processes, I don't see the problem here... Maybe I'm just completely blind to some detail. Would you be kind to clarify please?

The way I see it - we have one estimator per target. Say there are 2 target, we have `est1` and `est2`. And if you give `n_jobs>=2`. `est1` will `partial_fit` on process 1 and `est2` on process 2 correct? Both of these are cloned from the original est instance and hence should train on the required batch of data without any side effects from parallelism...